modules/gitops-configuring-namespace-scoped-argo-rollouts-installation.adoc

```diff
@@ -0,0 +1,110 @@
+// Module included in the following assemblies:
+//
+// * argo_rollouts/enable-support-for-namespace-scoped-argo-rollouts-installation.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-namespace-scoped-argo-rollouts-installation_{context}"]
+= Configuring a namespace-scoped Argo Rollouts installation
+
+To configure a namespace-scoped instance of Argo Rollouts installation, complete the following steps:
```

Comment:
Should really end in a period here. Not enough to hold up a merge review, but just calling it out. 

```suggestion
To configure a namespace-scoped instance of Argo Rollouts installation, complete the following steps.
```

Reply:
Fixed.